### DISCUSSION TITLE
`cmd+enter` in PR comment closes PR

### DISCUSSION THREAD

**Discussion by xdumaine** [community/community#19705] [Pull Requests] [answered]
This bug seems to have started today. I used to be able to quickly leave comments by typing, them pressing `cmd+enter`. Now when I do that, the PR gets closed.

![pr-bug](https://user-images.githubusercontent.com/833911/176494643-b56a89ef-a3de-429e-a6fa-2d86c580bdfa.gif)

**Comment by MarijnS95** [comment, marked as answer]
This was reported ±25min earlier in https://github.com/github-community/community/discussions/19701 too, feel free to upvote that one as well to get some attention to this :pray:

**Reply by xdumaine** [reply]
Adding a note that this has been resolved by github.
